code/modules/client/client_procs.dm

```diff
@@ -306,7 +306,8 @@ var/next_external_rsc = 0
 //DISCONNECT//
 //////////////
 
-/client/Del()
```

Comment:
so, sadly, I think there was a reason this was Del() and not Destroy. (that had to do with situations a client could disconnect that does not call mob/logout(because, to remind anybody reading this, mob login/logout has nothing to do with clients (dis)connecting)) Luckily the issue @Fox-McCloud warned us about only seems happens with normal datums, snowflake types like world or client do not seem to trigger the overhead